karspexet/settings.py

```diff
@@ -153,3 +153,4 @@
     exit(1)
 
 EMAIL_BACKEND = ENV.get("email_backend", 'django.core.mail.backends.smtp.EmailBackend')
+PAYMENT_PROCESS = ENV["payment_process"]
```

Comment:
Please user `ENV.get` so we don't blow up if it doesn't exist. If we should explode, then check it by hand and provide a nicer error message the KeyError